temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -234,3 +234,89 @@ message GetRegionResponse {
     // The temporal cloud region.
     temporal.api.cloud.region.v1.Region region = 1;
 }
+
+message GetNexusEndpointsRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100. 
+    int32 page_size = 1;
+
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+
+    // Filter endpoints based on their target specification - optional, treated as an AND if specified. Empty fields within the spec are treated as '*'
+    temporal.api.cloud.nexus.v1.EndpointTargetSpec = 3;
```

Comment:
delta from OSS: developers probably want ways to filter out endpoints based on either namespace, task queue or both. OSS doesn't have this right now. 

Other option is that we cut this for now and filter on client-side until a later point in time

Reply:
talked offline. we are good with keeping this for now